.github/workflows/test-lint-pr.yml

```diff
@@ -62,12 +62,19 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install uv
+        run: |
+          pip install uv
       - name: Install dependencies
         run: |
-          pip install --no-cache-dir hatch
+          uv venv
```

Comment:
This seems to be a direct port of the prior commands, but it seems like `uv` should be performing commands in the context of the venv automatically.

Specifically in the docs, they mention using the `uv` commands directly, like `sync`: https://docs.astral.sh/uv/guides/integration/github/#syncing-and-running - we shouldn't need to create or activate the virtual environment directly.

Similarly to run python, we should be using [`uv run`](https://docs.astral.sh/uv/reference/cli/#uv-run)